db-queue-spring/src/test/java/ru/yoomoney/tech/dbqueue/spring/dao/utils/H2DatabaseInitializer.java

```diff
@@ -0,0 +1,120 @@
+            "  pl    VARCHAR(100),\n" +
+            "  ct    TIMESTAMP WITH TIME ZONE DEFAULT now(),\n" +
+            "  pt    TIMESTAMP WITH TIME ZONE DEFAULT now(),\n" +
+            "  at    INTEGER                  DEFAULT 0,\n" +
+            "  rat   INTEGER                  DEFAULT 0,\n" +
+            "  tat   INTEGER                  DEFAULT 0,\n" +
+            "  trace TEXT \n" +
+            "); \n" +
+            "CREATE INDEX %s_name_time_desc_idx\n" +
+            "  ON %s (qn, pt, qid DESC);\n" +
+            "\n";
+
+    private static final String H2_DEFAULT_TABLE_DDL = "" +
+            "CREATE TABLE %s (\n" +
+            "  id                BIGSERIAL PRIMARY KEY,\n" +
+            "  queue_name        VARCHAR(100) NOT NULL,\n" +
+            "  payload           VARCHAR(100),\n" +
+            "  created_at        TIMESTAMP WITH TIME ZONE DEFAULT now(),\n" +
+            "  next_process_at   TIMESTAMP WITH TIME ZONE DEFAULT now(),\n" +
+            "  attempt           INTEGER                  DEFAULT 0,\n" +
+            "  reenqueue_attempt INTEGER                  DEFAULT 0,\n" +
+            "  total_attempt     INTEGER                  DEFAULT 0\n" +
+            ");\n" +
+            "CREATE INDEX %s_name_time_desc_idx\n" +
+            "  ON %s (queue_name, next_process_at, id DESC);\n" +
+            "\n";
+
+    private static final String H2_DEFAULT_WO_INC_TABLE_DDL = "CREATE TABLE %s (\n" +
+            "  id                BIGINT PRIMARY KEY,\n" +
+            "  queue_name        VARCHAR(100) NOT NULL,\n" +
+            "  payload           VARCHAR(100),\n" +
+            "  created_at        TIMESTAMP WITH TIME ZONE DEFAULT now(),\n" +
+            "  next_process_at   TIMESTAMP WITH TIME ZONE DEFAULT now(),\n" +
+            "  attempt           INTEGER                  DEFAULT 0,\n" +
+            "  reenqueue_attempt INTEGER                  DEFAULT 0,\n" +
+            "  total_attempt     INTEGER                  DEFAULT 0\n" +
+            ");\n" +
+            "CREATE INDEX %s_name_time_desc_idx\n" +
+            "  ON %s (queue_name, next_process_at, id DESC);\n" +
+            "\n";
+
+    private static JdbcTemplate h2JdbcTemplate;
+    private static TransactionTemplate h2TransactionTemplate;
+
+    public static synchronized void initialize() {
+        if (h2JdbcTemplate != null) {
+            return;
+        }
+
+        JdbcDataSource dataSource = new JdbcDataSource();
+        dataSource.setUrl("jdbc:h2:~/test");
+        h2JdbcTemplate = new JdbcTemplate(dataSource);
+        h2TransactionTemplate = new TransactionTemplate(new DataSourceTransactionManager(dataSource));
+        h2TransactionTemplate.setIsolationLevel(TransactionDefinition.ISOLATION_READ_COMMITTED);
+        h2TransactionTemplate.setPropagationBehavior(TransactionDefinition.PROPAGATION_REQUIRED);
+
+        h2JdbcTemplate.execute("DROP ALL OBJECTS DELETE FILES");
+        executeDdl("CREATE SEQUENCE tasks_seq START 1");
```

Comment:
Do we need it in all cases? If yes, then we must put it in README

Reply:
Nope, it's not needed in all cases. Only `DefaultH2WithSequenceQueueDaoTest` uses this sequences.